### PR TITLE
Release 17.1.0

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [17.1.0](https://github.com/theforeman/puppet-foreman/tree/17.1.0) (2021-05-04)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/17.0.0...17.1.0)
+
+**Implemented enhancements:**
+
+- Allow Puppet 7 compatible versions of mods [\#947](https://github.com/theforeman/puppet-foreman/pull/947) ([ekohl](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:9]: https://github.com/theforeman/puppet-candlepin/pull/194#pullrequestreview-651267371

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+- Allow customising the list of HTTP headers to unset [\#944](https://github.com/theforeman/puppet-foreman/pull/944) ([nbarrientos](https://github.com/nbarrientos))
+- Customisable Yum repository base URL and GPG key path [\#943](https://github.com/theforeman/puppet-foreman/pull/943) ([nbarrientos](https://github.com/nbarrientos))
```
Comment (ekohl) [inline CHANGELOG.md:11]: This turns out to break puppet-foreman_proxy: https://github.com/theforeman/puppet-foreman_proxy/issues/671

**Comment by wbclark** [comment]
Converted to draft to be picked up when needed

**Comment by ekohl** [comment]
We're doing a major version after this so I'm closing this.

**Pull request closed without merging**